homeassistant/components/zwave/__init__.py

```diff
@@ -240,24 +262,28 @@ def get_config_value(node, value_index, tries=5):
     from openzwave.network import ZWaveNetwork
     from openzwave.group import ZWaveGroup
 
+    config = {}
+    if DATA_ZWAVE_CONFIG in hass.data:
+        config = hass.data[DATA_ZWAVE_CONFIG]
+
     # Load configuration
-    use_debug = config[DOMAIN].get(CONF_DEBUG)
-    autoheal = config[DOMAIN].get(CONF_AUTOHEAL)
+    use_debug = config.get(CONF_DEBUG)
+    autoheal = config_entry.data.get(CONF_AUTOHEAL)
```

Comment:
Make sure that we don't store defaults in `data`. We should just apply them here: `config_entry.data.get(CONF_AUTOHEAL, DEFAULT_AUTOHEAL)`.

Also note, we don't have to put every option in the config entry. In fact, pretty sure we don't want to. Advanced options (like disabling autoheal) should be fetched from `config`. Every extra option we give a user is one too many 😉 If most don't need it, let's not show it.